### ISSUE TITLE
unable to recieve message on realtime basis

### ISSUE THREAD

**Issue by ankitrout2903** [ankitrout2903/Swingyy#1]
may be message.jsx or socket.js is problem if you can fix it

**Comment by 29deepanshutyagi** [comment]
Kindly assign me this issue under hactoberfest 2023 , i want to work on this issue

**Comment by ankitrout2903** [comment]
> Kindly assign me this issue under hactoberfest 2023 , i want to work on this issue

yes you can work on this problem kindly go through the repo, problem may be in client side messages.jsx or server side socket.js . also make sure to only give PR after 1st of october so that you get a valid PR.
Happy Hacking🫠
